dev/com.ibm.ws.transport.http/resources/com/ibm/ws/http/channel/internal/resources/httpchannelmessages.nlsprops

```diff
@@ -185,3 +185,11 @@ trusted.header.origin.invalid.value.useraction=Ensure that configured IP address
 trusted.sensitive.header.origin.invalid.value=CWWKT0039W: The [{0}] configured trustedSensitiveHeaderOrigin value is invalid and cannot be used.
 trusted.sensitive.header.origin.invalid.value.explanation=The specified value must be either a full IP address or correctly formatted hostname.
 trusted.sensitive.header.origin.invalid.value.useraction=Ensure that configured IP addresses contain a value for each segment, and that hostnames do not contain any invalid characters.
+
+config.hsts.invalid.max.age.value=CWWKT0040W: The [{0}] configured max-age value is invalid and cannot be used. The max-age value must be a non-negative integer.
+config.hsts.invalid.max.age.value.explanation=The value for max-age must be a non-negative integer.
+config.hsts.invalid.max.age.value.useraction=Ensure that the configured max-age directive contains a non-integer value.
```

Comment:
Ensure that the configured max-age directive contains a non-integer value.
--->
Ensure that the max-age directive contains a non-negative integer value.

Reply:
Made the recommended changes, thanks!